src/phasorpy/phasor.py

```diff
@@ -387,34 +390,37 @@ def phasor_semicircle(
     imag *= 0.5
     return real, imag
 
-
-def phasor_calibrate(
-    real: ArrayLike,
-    imag: ArrayLike,
-    phase0: ArrayLike = 0.0,
-    modulation0: ArrayLike = 1.0,
-    /,
-) -> tuple[NDArray[Any], NDArray[Any]]:
-    """Return calibrated/referenced phasor coordinates.
+def phasor_calibrate(real, imag, reference_real, reference_imag, frequency, lifetime, fraction, **kwargs):
+    """
+    Return calibrated/referenced phasor coordinates.
 
     Calibration of phasor coordinates in fluorescence lifetime analysis is
     necessary to account for the instrument response function (IRF) and delays
     in the electronics.
 
-    This function can also be used to transform/rotate any phasor coordinate.
-
     Parameters
     ----------
     real : array_like
-        Real component of phasor coordinates to calibrate.
+        Real component of phasor coordinates to be calibrated.
     imag : array_like
-        Imaginary component of phasor coordinates to calibrate.
-    phase0 : array_like, optional
-        Angular component of polar coordinates for calibration in radians.
-        Defaults to 0.0.
-    modulation0 : array_like, optional
-        Radial component of polar coordinates for calibration.
-        Defaults to 1.0.
+        Imaginary component of phasor coordinates to be calibrated.
+    reference_real : array_like
+        Real component of phasor coordinates from reference of known lifetime.
```

Comment:
Maybe add that the reference phasor coordinates must be measured at the same instrument settings as the phasor coordinates to be calibrated.